app/code/Magento/CatalogGraphQl/Model/Resolver/Products/DataProvider/CategoryTree.php

```diff
@@ -85,8 +86,8 @@ public function getTree(ResolveInfo $resolveInfo, int $rootCategoryId): \Iterato
     {
```

Comment:
@danielrenaud here in this existing method we have $resolveInfo as first param. Maybe I should better move it to the first for other methods?

Reply:
It's fine to leave all the parameters as you have them. It was just a suggestion to make the method signatures look more similar, but it's minor detail as these are not api methods or anything